src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/SalesforceSinkConfig.java

```diff
@@ -146,7 +153,8 @@ public SalesforceSinkConfig(String referenceName,
                               String errorHandling,
                               @Nullable String securityToken,
                               @Nullable OAuthInfo oAuthInfo,
-                              @Nullable String proxyUrl) {
+                              @Nullable String proxyUrl,
+                              @Nullable boolean datatypeValidation) {
```

Comment:
Nullable does not apply for primitive types. Did you mean it to be "Boolean"?

Reply:
no boolean is fine, I will remove it